proposals/0306-actors.md

```diff
@@ -77,7 +77,7 @@ actor BankAccount {
 }
 ```
 
-Like other Swift types, actors can have initializers, methods, properties, and subscripts. They can be extended and conform to protocols, be generic, and be used with generics.
+Like other Swift types, actors can have initializers, methods, properties, and subscripts. They cannot be extended, however can conform to protocols, be generic, and be used with generics.
```

Comment:
Ah, "extended" here means "you can write an extension of the type" not "you can create a subclass" in the Java sense.

Reply:
ah! That's true I suppose. Closing then